tests/test_counting_hash.py

```diff
@@ -323,7 +323,10 @@ def test_save_load():
     hi.save(savepath)
 
     ht = khmer.CountingHash(12, sizes)
-    ht.load(savepath)
+    try:
```

Comment:
OK. I don't regard a trade of four lines for one, with decreased readability and increased complexity, as a fair trade here...